contrib/gitian/gitian-build.py

```diff
@@ -7,21 +7,13 @@
 
 def setup():
     global args, workdir
-    programs = ['ruby', 'git', 'apt-cacher-ng', 'make', 'wget']
+    programs = ['ruby', 'git', 'make', 'wget']
```

Comment:
I would prefer to keep apt-cacher-ng here, and fix the docker image to use it. 
See https://github.com/devrandom/gitian-builder/pull/219
Otherwise the builds consume a huge amount of network bandwidth (and takes a much longer time.)